transforms/helpers/decorator-helper.js

```diff
@@ -46,30 +46,31 @@ function createCallExpressionDecorators(j, decoratorName, instanceProp) {
     return [];
   }
 
-  if (instanceProp.hasWrapComputedDecorator) {
-    return j.decorator(j.identifier(instanceProp.calleeName));
-  }
-
   const decoratorArgs =
     !instanceProp.hasMapDecorator &&
     !instanceProp.hasFilterDecorator &&
     instanceProp.shouldRemoveLastArg
       ? instanceProp.callExprArgs.slice(0, -1)
       : instanceProp.callExprArgs.slice(0);
 
-  const decoratorExpr = instanceProp.modifiers.reduce(
+  let decoratorExpression =
+    ['computed', 'service', 'controller'].includes(decoratorName) && decoratorArgs.length === 0
```

Comment:
Should we check against hard coded decorator names? Would it work in case they are imported with different local name? For example `service` is imported as `inject` or some other local name

Reply:
The `deocratorName` here is the name that it was imported _from_ not what is named in the module. I'll add a specific test case for that.